frontend/src/store/userStore.ts

```diff
@@ -31,7 +31,20 @@ export const useUserStore = defineStore('user', () => {
   // Getters
   const getters = {
     getIdps: computed(() => state.idps.value),
-    getUserSearch: computed(() => state.userSearch.value)
+    // returns a single User for provided user ID
+    getUser: computed(
+      () => (userId: string | undefined) => state.userSearch.value.find((x: User) => userId === x.userId)
+    ),
+    // returns an array of Users corresponding to provided user IDs
+    getUsers: computed(() => (userIds: Array<string> | undefined) => {
```

Comment:
Not absolutely certain, but assuming no typescript snafus, it might be better to have the blah | undefined be marked as blah? instead to signal optionality?